checkprovider

```diff
@@ -252,6 +273,16 @@ sub time_request($) {
         push @request, sprintf( "GET %s HTTP/1.1", $feed );
     }
     else {
+        # replace query term and save off term in stats hash
+        if ( defined $termsfile && $query =~ m/(\[TERM\])/ ) {
+            my $term = $terms[$position];
+            # try to encode the term if URI::Encode is available
+            if ( eval "use URI::Encode; 1" ) {
+                $term = (URI::Encode->new( { encode_reserved => 0 } ))->encode($terms[$position]);
```

Comment:
Oops forgot the last comment... fixed in the master branch: https://github.com/yahoo/checkprovider/commit/b661ca4